lib/dirac_improved_staggered_kd.cpp

```diff
@@ -231,19 +181,14 @@ namespace quda
     Dirac::updateFields(fat_gauge_in, nullptr, nullptr, nullptr);
     fatGauge = fat_gauge_in;
     longGauge = long_gauge_in;
-
-    // Recompute Xinv (I guess we should do that here?)
```

Comment:
What's the question here for?

Reply:
That was an old thought I had and have since discarded (or at least discarded implementing here).

The idea was you could imagine rebuilding _just_ `Xinv` when you update the fields as part of a "thin" MG update, since that's relatively inexpensive compared to a full rebuild of the MG preconditioner. I ended up discarded putting that in here because `updateFields` is essentially meant to update pointers, so slipping a "real" kernel in here would be inappropriate.

I might revisit the idea of rebuilding just `Xinv` in the future, but that'd be through some other interface and not here.